lib/x11test.pm

```diff
@@ -740,7 +740,9 @@ sub start_gnome_settings {
 
     if (!$is_sle_12_sp1 || $settings_menu_loaded) {
         assert_and_click 'settings';
-        assert_screen 'gnome-settings';
+        my $timeout = 30;
+        $timeout = 180 if (check_var('ARCH', 'aarch64'));
```

Comment:
Only aarch64?

Reply:
This issue only happened on ARM so we shouldn't enlarge the timeout value for other arches when the value is much larger than the original value, or this will lead to need wait unexpected time when other issue happens for other arches.